code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -119,7 +109,7 @@
 	allowed_roles = list("Chief Medical Officer", "Medical Doctor", "Pharmacist","Psychiatrist", "Paramedic", "Medical Resident")
 
 /datum/gear/accessory/pouches
-	display_name = "drop pouches, simple"
+	display_name = "drop pouches"
```

Comment:
See above for why 'simple' should remain even if you're removing the job-specific ones.